frontend/packages/knative-plugin/src/components/knative-resource/OverviewDetailsKnativeResourcesTab.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+
+import { K8sResourceKind } from '@console/internal/module/k8s';
+import { SidebarSectionHeading } from '@console/internal/components/utils';
+import { OverviewItem } from '@console/internal/components/overview';
+import ConfigurationsOverviewList from './ConfigurationsOverviewList';
+import RevisionsOverviewList from './RevisionsOverviewList';
+import KSRoutesOverviewList from './KSRoutesOverviewList';
+
+type KnativeOverviewProps = {
+  ksroutes: K8sResourceKind[];
+  configurations: K8sResourceKind[];
+  revisions: K8sResourceKind[];
+};
+
+type OverviewDetailsResourcesTabProps = {
```

Comment:
You should export the types of the components that are also exported.
```suggestion
export type OverviewDetailsResourcesTabProps = {
```

Reply:
We are exporting component as default export, should we name export for both?